site/en/docs/extensions/reference/declarativeNetRequest/index.md

```diff
@@ -281,140 +322,90 @@ When rules are applied to browsers with pages in the service worker's cached sto
     "action": { "type": "redirect", "redirect": { "url": "https://example.com" } },
     "condition": { "urlFilter": "google.com", "resourceTypes": ["main_frame"] }
   },
-  {
-    "id": 5,
-    "priority": 1,
-    "action": { "type": "redirect", "redirect": { "extensionPath": "/a.jpg" } },
-    "condition": { "urlFilter": "abcd.com", "resourceTypes": ["main_frame"] }
-  },
-  {
-    "id": 6,
-    "priority": 1,
-    "action": {
-      "type": "redirect",
-      "redirect": {
-        "transform": { "scheme": "https", "host": "new.example.com" }
-      }
-    },
-    "condition": { "urlFilter": "||example.com", "resourceTypes": ["main_frame"] }
-  },
-  {
-    "id": 7,
-    "priority": 1,
-    "action": {
-      "type": "redirect",
-      "redirect": {
-        "regexSubstitution": "https://\\1.xyz.com/"
-      }
-    },
-    "condition": {
-      "regexFilter": "^https://www\\.(abc|def)\\.xyz\\.com/",
-      "resourceTypes": [
-        "main_frame"
-      ]
-    }
-  },
-  {
-    "id" : 8,
-    "priority": 2,
-    "action" : {
-      "type" : "allowAllRequests"
-    },
-    "condition" : {
-      "urlFilter" : "||b.com/path",
-      "resourceTypes" : ["sub_frame"]
+]
+```
+
+#### Redirects {: #redirects }
+
+To use the example file shown below, your manifest file would need the following host permission:
+
+`*://*.example.com/*`
+
+The following example shows how to redirect a request from example.com to a page within the extension itself. The extension path `/a.jpg` resolves to `chrome-extension://EXTENSION_ID/a.jpg`, where `EXTENSION_ID` is the ID of your extension.
```

Comment:
It might be a good idea to mention that the a.jpg also needs to be declared as a WAR. Just in case a developers skipped the WAR section.

Reply:
Done.